docs/docs/reach-router-and-gatsby.md

```diff
@@ -2,7 +2,22 @@
 title: "@reach/router and Gatsby"
 ---
 
-This is a stub. Help our community expand it.
+This guide gives a peek under the hood of using `@reach/router` with Gatsby.
 
-Please use the [Gatsby Style Guide](/contributing/gatsby-style-guide/) to ensure your
-pull request gets accepted.
+## Why do we use @react/router?
```

Comment:
You'll want to change this to say `@reach/router` rather than `@react/router`.

Reply:
😂 Thank you